lib/tinify/source.rb

```diff
@@ -9,6 +9,11 @@ def from_buffer(string)
         response = Tinify.client.request(:post, "/shrink", string)
         new(response.headers["Location"]).freeze
       end
+
+      def from_url(url)
+        response = Tinify.client.request(:post, "/shrink", { "source" => { "url" => url } })
```

Comment:
I think we use symbol hashes in other places, so this could be `(:post, "/shrink", source: { url: url })`